### PR TITLE
feat: param changes for block and cost per byte

### PR DESCRIPTION
**Pull request by PaddyMc**
## What is the purpose of the change

Param changes for block size and cost per byte

### Analysis

1. with the tx_size_cost_per_byte=20 rn, a swap from osmo => usdc through pool 1464 costs => 0.0003c
2. continuing if we 10x the value of osmo, swaps cost => 0.003c
3. with the tx_size_cost_per_byte=30 change the same swap goes to => 0.0004c
4. continuing if we 10x the value of osmo, swaps cost => 0.004c

**TLDR**: I think we're good to increase cost per byte from 20 => 30

Average blocks 
```
Block 18679972: Size 0.059330 MB
Block 18679973: Size 0.087567 MB
Block 18679974: Size 0.037231 MB
Block 18679975: Size 0.067585 MB
Block 18679976: Size 0.042751 MB
Block 18679977: Size 0.086765 MB
Block 18679978: Size 0.069870 MB
Block 18679979: Size 0.067013 MB
Block 18679980: Size 0.120811 MB
Block 18679981: Size 0.037151 MB

```
### How to test 

- test with in-place-testnet and
- `curl http://localhost:1317/cosmos/consensus/v1/params | jq`
```
{
  "params": {
    "block": {
      "max_bytes": "2000000",
      "max_gas": "300000000"
    },
    "evidence": {
      "max_age_num_blocks": "100000",
      "max_age_duration": "172800s",
      "max_bytes": "1048576"
    },
    "validator": {
      "pub_key_types": [
        "ed25519"
      ]
    },
    "version": {
      "app": "0"
    },
    "abci": {
      "vote_extensions_enable_height": "0"
    }
  }
}

```

- `osmosisd q auth params --output json | jq `

```
{
  "params": {
    "max_memo_characters": "256",
    "tx_sig_limit": "7",
    "tx_size_cost_per_byte": "30",
    "sig_verify_cost_ed25519": "590",
    "sig_verify_cost_secp256k1": "1000"
  }
}

```

### CODE REVIEW

**Review by czarcas7ic — APPROVED** [review]
Nice, where were these values picked from btw?

**Comment by PaddyMc** [comment]
> Nice, where were these values picked from btw?

Blocksize:
The 2mb was based on a discussion after looking at benchmarks more details here => https://linear.app/osmosis/issue/CHAIN-603/update-max-bytes-param-in-consensus-param-to-be-2mb

CostPerByte:
I need to do more investigation here, this was just a hand wavy estimate, will add more details to the PR description today

File: app/upgrades/v26/constants.go [hunk 1]
```diff
@@ -13,6 +13,15 @@ const (
 
 	// MaximumUnauthenticatedGas for smart account transactions to verify the fee payer
 	MaximumUnauthenticatedGas = uint64(250_000)
+
+	// BlockMaxBytes is the max bytes for a block, 2mb
```
Comment (ValarDragon) [inline app/upgrades/v26/constants.go:17]:
```suggestion
	// BlockMaxBytes is the max bytes for a block, 3mb
```

**Review by ValarDragon — APPROVED** [review]
LGTM!

**Pull request merged**